src/styles/_catalog.scss

```diff
@@ -1,22 +1,103 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$margin-reset: 0;
 
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 48px;
+  justify-items: center;
   max-width: 944px;
   margin: 0 auto;
+  padding-bottom: 50px;
+
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @media (max-width: 767px) {
+    grid-template-columns: repeat(2, 1fr);
+  }
```

Comment:
It is not specified explicitly in the task, but I think you should have a one-row layout for mobile devices as well. At the moment your page has a 2-row layout for mobile devices, and even though the cards fit, they don't have any padding around them which makes the page look unbalanced. 

I suggest adding one more media query that will make cards stand in one row on mobile.

![image](https://user-images.githubusercontent.com/67225216/161055702-b27b425f-d951-4202-b3c2-862bf13c48b4.png)

Reply:
I could do that but the reason I did not do it in the first place is because the header breaks at mobile-screen width, and we aren't suppose to touch it

Reply:
this is not a responsive design

Reply:
i mean this one, does not have to be resposive